### PR TITLE
Fix ui cases

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei  please help to review, thanks!
fix OCP-21270,OCP-25795,OCP-20932,OCP-19558,OCP-26170,OCP-27584

Fixed below failed cases in automation run: 
```cucumber -p polarshift features/tierN/web/admin-console/overview.feature:201 # Scenario: Check stateful set on overview page
cucumber -p polarshift features/tierN/web/admin-console/overview.feature:386 # Scenario: Group resources
cucumber -p polarshift features/tierN/web/admin-console/deployments.feature:184 # Scenario: Check basic process of create/delete DC from image
cucumber -p polarshift features/tierN/web/admin-console/build.feature:99 # Scenario: Check deprecation note of pipeline build strategy
cucumber -p polarshift features/tierN/web/admin-console/overview.feature:5 # Scenario: Check app resources on overview page
cucumber -p polarshift features/tierN/web/admin-console/deployments.feature:5 # Scenario: Check deployment page
cucumber -p polarshift features/tierN/web/admin-console/route.feature:207 # Scenario: Support create route with multiple services
cucumber -p polarshift features/tierN/web/admin-console/dashboards.feature:6 # Scenario: Add view all alerts to status card```

### CODE REVIEW

**Comment by SSshahan** [comment]
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/87939/console 
OCP-19558 failed on above job, below job passed after fixing

File: lib/rules/web/admin_console/4.1/actions.xyaml [hunk 1]
```diff
@@ -16,6 +16,11 @@ click_resource_action_icon:
   - selector:
       xpath: //a[text()='<resource_name>']/ancestor::span/../div//span[contains(@class,'co-m-cog__icon')]
     op: click
+edit_pod_count_from_dropdown:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.1/actions.xyaml:19]: just `edit_pod_count` is enough, `from_xxx` is limited

**Pull request merged**